src/igvfd/types/sample.py

```diff
@@ -92,7 +92,8 @@ class Sample(Item):
              '@id', 'summary', 'status', 'tagged_proteins', 'modality', 'fused_domain', 'symbol', 'cas', 'cas_species', 'degron_system']),
         Path('institutional_certificates', include=['@id', 'certificate_identifier', 'status']),
         Path('construct_library_sets.associated_phenotypes', include=[
-             '@id', 'accession', 'file_set_type', 'term_name', 'associated_phenotypes', 'status'])
+             '@id', 'accession', 'file_set_type', 'term_name', 'associated_phenotypes', 'status']),
+        Path('donors', include=['@id', 'accession', 'strain', 'ethnicities']),
```

Comment:
Could you add an embed for the `status` property in case we display these in a table?

Reply:
yes